static/js/settings_realm_user_settings_defaults.js

```diff
@@ -20,5 +22,32 @@ export function set_up() {
     const container = $("#realm-user-default-settings");
     settings_display.set_up(container, realm_user_settings_defaults, true);
     settings_notifications.set_up(container, realm_user_settings_defaults, true);
+
+    container.find(".presence_enabled").on("change", (e) => {
+        e.preventDefault();
+        e.stopPropagation();
+
+        const data = {presence_enabled: container.find(".presence_enabled").prop("checked")};
+        settings_ui.do_settings_change(
+            channel.patch,
+            "/json/realm/user_settings_defaults",
+            data,
+            container.find(".privacy-setting-status").expectOne(),
+        );
+    });
+
+    container.find(".enter_sends").on("change", (e) => {
+        e.preventDefault();
+        e.stopPropagation();
+
+        const data = {enter_sends: container.find(".enter_sends").prop("checked")};
+        settings_ui.do_settings_change(
+            channel.patch,
+            "/json/realm/user_settings_defaults",
+            data,
+            container.find(".other-setting-status").expectOne(),
+        );
+    });
```

Comment:
Have added separate handlers for these settings, since these are in different files and do not belong to either display or notification settings. There is duplication but I feel it is cleaner.

Reply:
I think this is reasonable for now; we probably long term will end up doing something more like having a `privacy_settings.js` with similar structure, but it's not clear it's worth the effort to refactor for that at this juncture.  (Since we don't totally know what we will want until we have 5 more settings in that class).